### ISSUE TITLE
Add window management features (resize, snap to sides, etc)

### ISSUE THREAD

**Issue by dazeq** [lwouis/alt-tab-macos#1136]
Add a feature like the Window Management in HyperDock, don't be too complicated, just be simple. snap the windows or use shortcuts to resize.

**Comment by lwouis** [comment]
I must agree that it would be nice to use AltTab to quickly set 2 windows side-by-side for instance. That being said, I'm not sure:

* If AltTab should take on that scope, or leave it to other apps
* What UX would be best? I don't think I want to support complex scenarios as dedicated apps would do it better. So if it's only supporting side-by-siding 2 apps, there is probably a pretty nice UX to invent there

**Comment by lwouis** [comment]
I've decided that these are out-of-scope for AltTab. AltTab focus will remain switching windows.
